### ISSUE TITLE
Uncaught TypeError: Cannot read property 'dispatchEvent' of null

### ISSUE THREAD

**Issue by steve13814** [novnc/noVNC#779]
I'm using novnc master branch (commit a7ca8e5). When I connect to a ubuntu vm by Chrome 53.0.2785.34 and do some mouse clicks. The "Uncaught TypeError" happend.
![screenshot_20170223_180352](https://cloud.githubusercontent.com/assets/502278/23254692/e3dcbcc0-f9f3-11e6-87f5-7fd2f8eee77e.png)

At first, this issue is hard to reproduce. I found that after adding some busy waiting, this bug is reproducible by keep clicking the mouse on the canvas

```diff
diff --git a/core/util.js b/core/util.js                                                                                           
index 03e6925..e374dd2 100644                                                                                                      
--- a/core/util.js                                                                                                                 
+++ b/core/util.js                                                                                                                 
@@ -564,6 +564,11 @@ Util.setCapture = function (elem) {                                                                           
             captureElem.addEventListener('touchend', Util._captureProxy);                                                         
         }                                                                                                                         
+       var date = new Date();                                                                                                     
+       var curDate = null;                                                                                                        
+       do { curDate = new Date();  }                                                                                              
+       while(curDate-date < 200);                                                                                                 
+                                                                                                                                  
         Util._captureElem = elem;                                                                                                 
         // Track cursor and get initial cursor
```

**Comment by CendioOssman** [comment]
This should have been fixed in 1658466579ee638aa29aaa7adfc18bb64348737c. Have you tried clearing your cache?

**Comment by steve13814** [comment]
Hi~ I didn't use any old version of novnc.
Anyway, I just deleted chrome local storage, cookies. Open the Chrome development tool and check the option to disable the cache.
As long as I add some busying waiting code, this bug is reproducible.

**Comment by CendioOssman** [comment]
Great. I'll have a look and see if I can reproduce it here as well.

**Comment by CendioOssman** [comment]
Fixed in 90ecc73.
